<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiHeaderDialog</class>
 <widget class="QWidget" name="uiHeaderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_54">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>9</y>
     <width>179</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Database Records</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="uiRecords">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>34</y>
     <width>599</width>
     <height>325</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabHeader">
    <attribute name="title">
     <string>Header</string>
    </attribute>
    <widget class="QWidget" name="uiLayoutGeometry">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>551</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridHeader" rowstretch="1,1,1,1,5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="uiVersion">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_19">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="uiFile">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Name of Product</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="uiProductName">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_21">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Gedcom vers</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="uiGedComVers">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="uiDate">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_22">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Gedcom form</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="uiGedComForm">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_23">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Character Set</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="uiCharSet">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QTextEdit" name="uiNotes"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="uiTime">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSubmitter">
    <attribute name="title">
     <string>Submitter</string>
    </attribute>
    <widget class="QDateTimeEdit" name="uiSubmChanged">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="keyboardTracking">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="uiLayoutGeometry2">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>20</y>
       <width>513</width>
       <height>197</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridSubmitter">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_57">
        <property name="text">
         <string>Name / Language</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_36">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>address line</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLineEdit" name="uiSubmAddr"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_52">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>address line 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="uiSubmAddr1"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_51">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>address line 2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLineEdit" name="uiSubmAddr2"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_50">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>PO / City</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="uiSubmCity"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_55">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>State / Country</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="uiSubmPost"/>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="uiSubmState"/>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="uiSubmCountry"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="uiSubmName"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="uiSubmLang"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="uiLayoutGeometry5">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>366</y>
     <width>158</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="uiExit">
      <property name="text">
       <string>Save &amp; Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
